packages/fluentui/react-northstar/src/components/Design/Design.tsx

```diff
@@ -18,6 +18,7 @@ export type DesignProps = {
 /**
  * The Design component provides a theme safe subset of CSS for designing layouts.
  */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Can you please clarify why it screams there? What is unused?

Reply:
IIRC it said DesignProps is unused. If you think it's safe to remove the type parameter I can just do that instead.

Reply:
Let's keep it for now, I will fix it later 👍